study-builder/studies/brain/emails/exit_request.html

```diff
@@ -75,7 +75,7 @@
   <tr style="width: 540px;">
     <td align="left" style="margin: 0; padding: 0; font-family: Roboto, sans-serif; width: 278px; height: 33px">
       <a href="-ddp.baseWebUrl-" target="_blank" style="cursor: pointer; text-decoration: none;">
-        <img src="https://storage.googleapis.com/brain-prod-assets/BrainCancerHeader.png"
+        <img src="https://storage.googleapis.com/cmi-study-dev-assets/brain/project-logo.png"
```

Comment:
why is this pointing to dev-assets rather than variable substitution like "${assetsBucketName}"/brain ??
With pointing to cmi-study-dev-assets, production will be looking at dev bucket for this resources. right!

Reply:
ok. This is just emails checked into git. got it.